src/server/index.js

```diff
@@ -91,6 +98,11 @@ if (fs.existsSync(customConfigPath)) {
         ...customConfig.module.loaders || [],
       ],
     },
+    resolve: {
```

Comment:
Now we allow to configure the `resolve` API directly via [custom webpack API](https://github.com/kadirahq/react-storybook/blob/master/docs/api.md#custom-webpack-configurations).

In this case, we simply need to document this.
What do you think?